packages/typed-openapi/tstyche.config.json

```diff
@@ -1,8 +1,7 @@
 {
   "$schema": "https://tstyche.org/schemas/config.json",
   "checkSuppressedErrors": true,
-  "checkSourceFiles": true,
-  "tsconfig": "./tsconfig.ci.json",
+  "checkSourceFiles": false,
```

Comment:
The `checkSourceFiles` option is removed in the next major. It does way too much.

`tsconfig.ci.json` is not picked, because it does not include the test file. TSTyche default are used instead a they are [rather strict](https://tstyche.org/project/compiler-options#default-compiler-options).

Without `tsconfig` option set, `packages/typed-openapi/tsconfig.json` is loaded. I think that is right, or? Look for 'uses' line to see which TSConfig is loaded ([documentation](https://tstyche.org/project/compiler-options#default-compiler-options)):

<img width="360" height="56" alt="Image" src="https://github.com/user-attachments/assets/8e356a93-552d-4aa3-a256-a3f74c945b83" />

Reply:
> btw how did you find out that I started using TSTyche?

GitHub told me: https://github.com/tstyche/tstyche/network/dependents

(;